microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -1302,6 +1305,360 @@
+                .attr("x", x)
+                .attr("y", y)
+                .attr("transform", `rotate(${(tangentAngle * 180) / Math.PI + angleOffset}, ${x}, ${y})`);
+        }
+
+        if(diversityInSampleValue >= 6.5 && diversityInSampleValue < 6.99){
+            var scale = d3.scaleLinear()
+                .domain([2.40, 7.25])
+                .range([Math.PI * 0.77, Math.PI * 2.40]);
+
+            var angle = scale(diversityInSampleValue);
+
+            var ellipseXRadius = radius * 1.10;
+            var ellipseYRadius = radius * 0.55;
+
+            var x = Math.cos(angle) * ellipseXRadius;
+            var y = Math.sin(angle) * ellipseYRadius;
+
+            var tangentAngle = Math.atan2(
+                ellipseYRadius * Math.cos(angle),
+                -ellipseXRadius * Math.sin(angle)
+            );
+            var angleOffset = -35;
+            svg.append("image")
+                .attr("xlink:href", "/static/img/Line_40.svg")
+                .attr("x", x)
+                .attr("y", y)
+                .attr("transform", `rotate(${(tangentAngle * 180) / Math.PI + angleOffset}, ${x}, ${y})`);
+        }
+
+        if(diversityInSampleValue >= 7.0 && diversityInSampleValue < 7.25){
+            var scale = d3.scaleLinear()
+                .domain([2.40, 7.25])
+                .range([Math.PI * 0.75, Math.PI * 2.41]);
+
+            var angle = scale(diversityInSampleValue);
+
+            var ellipseXRadius = radius * 1.65;
+            var ellipseYRadius = radius * 0.55;
+
+            var x = Math.cos(angle) * ellipseXRadius;
+            var y = Math.sin(angle) * ellipseYRadius;
+
+            var tangentAngle = Math.atan2(
+                ellipseYRadius * Math.cos(angle),
+                -ellipseXRadius * Math.sin(angle)
+            );
+            var angleOffset = -50;
+            svg.append("image")
+                .attr("xlink:href", "/static/img/Line_40.svg")
+                .attr("x", x)
+                .attr("y", y)
+                .attr("transform", `rotate(${(tangentAngle * 180) / Math.PI + angleOffset}, ${x}, ${y})`);
+        }
+
+        svg.append("image")
+            .attr("xlink:href", "/static/img/Line_38_blue.svg")
+            .style("transform", "rotate(49deg)")
+            .attr("x", 35)
+            .attr("y", -radius + 96.5);
+
+        svg.append("image")
+            .attr("xlink:href", "/static/img/Line_38_blue.svg")
+            .style("transform", "rotate(-18deg)")
+            .attr("x", 43)
+            .attr("y", -radius + 121);
+
+        svg.append("text")
+            .attr("x", -radius - 50)
+            .attr("y", 145)
+            .style("fill", "#747678")
+            .style("font-size", "14px")
+            .style("line-height", "24px")
+            .style("font-family", "Mulish")
+            .style("font-weight", "400")
+            .style("font-style", "normal")
+            .text("Lowest in our dataset");
```

Comment:
All static text needs to be exposed to the translation mechanism. (e.g. `.text("{{ _('Lowest in our dataset') }}");` )